easybuild/easyconfigs/s/SEPP/SEPP-4.3.10-foss-2019b.eb

```diff
@@ -0,0 +1,42 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
```

Comment:
```suggestion

easyblock = 'PythonPackage'

```

Reply:
No, this requires https://github.com/easybuilders/easybuild-easyblocks/pull/1998